canvas_modules/harness/cypress/support/canvas/node-cmds.js

```diff
@@ -603,3 +603,19 @@ Cypress.Commands.add("resizeNode", (nodeLabel, corner, newWidth, newHeight) => {
 			cy.resizeObjectToDimensions(srcBodySelector, srcSizingSelector, corner, newWidth, newHeight);
 		});
 });
+
+Cypress.Commands.add("tabNdodeAndAddWithSpaceBar", (nodeLabel, categoryLabel) => {
+	cy.findNodeInCategory(nodeLabel, categoryLabel)
+	// This code simulates the user pressing tab to move the keyboard focus.
+	// TODO - Use the Cypress tab() method when "Native Events" are supported in Cypress
+		.focus()
+		.type(" ");
+});
+
+Cypress.Commands.add("tabCategoryAndOpenWithSpaceBar", (categoryLabel) => {
+	cy.findCategory(categoryLabel)
+	// This code simulates the user pressing tab to move the keyboard focus.
+	// TODO - Use the Cypress tab() method when "Native Events" are supported in Cypress
+		.type(" ")
+		.focus();
+});
```

Comment:
If you play around with this you will see that the `.focus()` calls in the two tests above are not actually adding anything because the `.type(" ")`  seems to move focus to the object AND simulate a space bar key press on the object.  Consequently I would remove the `.focus()` calls  but leave in the comment to say that this is simulating a tab() press.

Reply:
.focus() has been removd.